sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CommunicationIdentifier.java

```diff
@@ -8,4 +8,9 @@
  * Common Communication Identifier for Communication Services
  */
 public abstract class CommunicationIdentifier {
+    /**
+     * Get string representation of the identifier
```

Comment:
It's important that we explain that this is the **full** id.

```suggestion
     * Full id of the identifier
```